falcon/request.py

```diff
@@ -683,6 +692,14 @@ def access_route(self):
     def remote_addr(self):
         return self.env.get('REMOTE_ADDR')
 
+    @property
+    def port(self):
```

Comment:
In uri attribute, if port is not coming from HTTP_HOST port is taken from SERVER_PORT.
[https://github.com/falconry/falcon/blob/master/falcon/request.py#L553](https://github.com/falconry/falcon/blob/master/falcon/request.py#L553)